src/main/java/uk/gov/cshr/config/SecurityConfig.java

```diff
@@ -65,14 +55,22 @@ public void globalUserDetails(IdentityService identityService, AuthenticationMan
     protected void configure(HttpSecurity http) throws Exception {
 
         http
-            .antMatcher("/**")
-            .authorizeRequests()
-                .antMatchers("/", "/login**", "/webjars/**", "/oauth/token", "/management/**")
+                .antMatcher("/**")
+                .authorizeRequests()
+                .antMatchers("/", "/login**", "/webjars/**", "/oauth/token")
                 .permitAll()
-            .anyRequest()
+                .anyRequest()
                 .authenticated()
-            .and().exceptionHandling()
-                .authenticationEntryPoint(new LoginUrlAuthenticationEntryPoint("/"));
+                .and()
+                .formLogin()
+                .loginPage("/login")
+                .defaultSuccessUrl("/management")
+                .failureUrl("/login?error=true")
+                .and()
+                .logout()
+                .logoutSuccessUrl("/login")
+                .and().exceptionHandling()
+                .authenticationEntryPoint(new LoginUrlAuthenticationEntryPoint("/login"));
```

Comment:
You'll need to rebase and update this taking into account new port restrictions.